examples/se3_sam.cpp

```diff
@@ -380,7 +382,7 @@ int main()
 
             // store
             poses_simu. push_back(X_simu);
-            poses.      push_back(Xi + SE3Tangentd::Random()); // use very noisy priors
+            poses.      push_back(Xi + (SE3Tangentd::Random()*0.1)); // use very noisy priors
```

Comment:
remove the "very noise priors" comment if using 0.1*rand

Reply:
Fix : `// use noisy priors`.